drivers/spi/spi_smartbond.c

```diff
@@ -106,13 +114,41 @@ static inline int spi_smartbond_set_word_size(const struct spi_smartbond_cfg *cf
 	return 0;
 }
 
+static inline void spi_smartbond_pm_policy_state_lock_get(struct spi_smartbond_data *data)
+{
+#if defined(CONFIG_PM_DEVICE_RUNTIME)
```

Comment:
This should be changed to `CONFIG_PM_DEVICE`

Reply:
fixed